openapi-spec/swagger.yaml

```diff
@@ -3130,155 +3242,623 @@ paths:
-            nano: 14000000
-            hour: 14
-            minute: 36
-            second: 58
-    DataStorageLoS:
-      type: object
-      properties:
-        recoveryTimeObjective:
+      - name: limit
+        in: query
+        description: Number of query results on multiple pages
+        schema:
+          minimum: 1
           type: integer
-          format: int64
-        provisioningPolicy:
-          type: string
-          enum:
-          - Thin
-          - Thick
-        isSpaceEfficient:
-          type: boolean
-        characterCodeSet:
-          type: string
-        maxFileNameLengthBytes:
+          format: int32
+      - name: start
+        in: query
+        description: Start position of the pagination query
+        schema:
+          minimum: 0
           type: integer
-          format: int64
-        storageAccessCapability:
+          format: int32
+      - name: sortKey
+        in: query
+        description: Sorting keyword, which can be sorted based on the number of initiators
+        schema:
           type: string
           enum:
-          - Read
-          - Write
-          - Execute
-      description: DataStorageLoS can be used to describe a service option covering
-        storage provisioning and availability.
-    IOConnectivityLoS:
-      required:
-      - accessProtocol
-      type: object
-      properties:
-        accessProtocol:
+          - initiatorCount
+          - volumeCount
+      - name: sortDir
+        in: query
+        description: Ascending or descending by size
+        schema:
           type: string
           enum:
-          - iscsi
-          - rbd
-          - fibre_channel
-          - DSWARE
-        maxIOPS:
-          type: integer
-          format: int64
-        maxBWS:
-          type: integer
-          format: int64
-      description: IOConnectivityLoS can be used to specify the characteristics of
-        storage connectivity.
-    DataProtectionLoS:
-      type: object
-      properties:
-        isIsolated:
-          type: boolean
-        minLifetime:
-          type: string
-          example: P3Y6M4DT12H30M5S
-        RecoveryGeographicObjective:
+          - asc
+          - des
+      - name: name
+        in: query
+        description: Host name
+        schema:
+          minLength: 1
+          maxLength: 255
           type: string
-          enum:
-          - Datacenter
-          - Rack
-          - RackGroup
-          - Region
-          - Row
-          - Server
-        RecoveryPointObjectiveTime:
+      responses:
+        200:
+          description: OK
+          content:
+            application/json:
+              schema:
+                type: array
+                items:
+                  $ref: '#/components/schemas/HostRespSpec'
+        401:
+          description: NotAuthorized
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/ErrorSpec'
+        403:
```

Comment:
Any specific reasons to include 403 to Host resources. Does other sensitive resources also requires 403?